routing/RouteSolver.py

```diff
@@ -0,0 +1,134 @@
+        anotherNode = True
+        while anotherNode:
+            nodeName = input("Please enter connected node name for node {}:\n".format(node))
+            if(nodeName in graph.vertices()):
+                while True:
+                    direction = input("Please enter the cardinal direction traveling from {} to {}: (N E S W)\n".format( node, nodeName))
+                    if direction is "N" or "S" or "E" or "W":
+                        graph.add_edge((node,nodeName))
+                        anotherConnection = input("Is there another connected node? (y/n)\n")
+                        if(anotherConnection == 'n'):
+                            anotherNode = False
+                        elif(anotherConnection == 'y'):
+                            #do nothing
+                            anotherNode = True
+                        else:
+                            while True:
+                                print("Invalid Input Given.")
+                                anotherConnection = input("Is there another connected node? (y/n)\n")
+                                if(anotherConnection == 'n'):
+                                    anotherNode = False
+                                elif(config == 'y'):
+                                    break
+                        break
+                    else:
+                        print("Invalid direction given. Please enter N, E, S, or W")
+            else:
+                connected = input("Invalid node name given. Is there a node connected? (y/n)\n")
+                if(connected == 'n'):
+                    anotherNode = False
+                elif(connected == 'y'):
+                    anotherNode = True
+
+def main(argv):
+    if len(argv)>0:
+        loadConfig(argv[0])
+    else:
+        getGraphDetailsFromUser()
+    while True:
+        startNode = input("Please enter the start node:\n")
+        if(startNode in graph.vertices()):
+            break;
+        else:
+            print("Invalid start node provided\n")
+    while True:
+        endNode = input("Please enter the end node:\n")
+        if(startNode in graph.vertices()):
+            break;
+        else:
+            print("Invalid end node provided\n")
+    paths = graph.find_path(startNode,endNode)
+    print("This is a single path: ")
+    print(paths)
+    allPaths = graph.find_all_paths(startNode,endNode)
+    print("These are all possible paths: ")
+    print(allPaths)
+    for p in allPaths:
+        print(p)
+
+##################################################################################
+##                                                                              ##
+##                            Exeutable code below                              ##
+##                                                                              ##
+##################################################################################
+if __name__ == "__main__":
+	main(sys.argv[1:])
```

Comment:
use `argparse` instead of `sys`

Reply:
Consider defining an option such as `-c, --config` that takes in the config file name.